app/helpers/application_helper/button/basic.rb

```diff
@@ -12,6 +12,10 @@ def initialize(view_context, view_binding, instance_data, props)
     end
   end
 
+  def check_rbac?
```

Comment:
Yes we discussed that. But then @romanblanco invesigated and found some exceptional behavior.

So we will follow the same pattern. Have the simple test in the base class and more complex or exceptional behavior will be implemented in `Button` subclasses.

That way the exceptional behavior will be encapsulated with the buttons that need that behavior and not in general code that handles all the buttons.